### PR TITLE
Bootstrap system using pacstrap instead of extracting a rootfs tarball

### PR DESCRIPTION
**Pull request by raccube**
This also adds our kit-packages repo and installs srobo-kit

### CODE REVIEW

File: stage1/00-init.sh [hunk 1]
```diff
@@ -13,3 +13,7 @@ pacman-key --populate archlinuxarm
 
 info "Updating system"
 pacman -Syu --noconfirm
+
+info "Setting up user accounts"
+useradd -mp $(openssl passwd -crypt tobor) robot
```
Comment (trickeydan) [inline stage1/00-init.sh]: Is openssl a dep here? Can we use chpasswd instead? Maybe even just a static hash?

File: stage0/pacman.conf [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+#NoUpgrade   =
+#NoExtract   =
+
+# Misc options
+#UseSyslog
+#Color
+#TotalDownload
+CheckSpace
+#VerbosePkgLists
+ILoveCandy
```
Comment (trickeydan) [inline stage0/pacman.conf:37]: que?

**Review by trickeydan — APPROVED** [review]
Nice work!

File: stage0/packages [hunk 3]
```diff
@@ -0,0 +1,3 @@
+base
+openssh
+srobo-kit
```
Comment (trickeydan) [inline stage0/packages:3]: We may want to add this package later btw

File: stage0/pacman.conf (hunk 2 above)
Comment (raccube) [reply stage0/pacman.conf:37]: Spices up progress bars

Comment (raccube) [reply stage0/pacman.conf:37]: Looks like this:
image

**Pull request merged**